guides/plugins/plugins/administration/using-vuex-state.md

```diff
@@ -0,0 +1,102 @@
+# Using Vuex Stores
+
+The Shopware 6 Administration uses [Vuex](https://vuex.vuejs.org/) stores to keep track of complex state, while just adding a wrapper around it.
+Learn what Vuex is, how to use it and when to use it from their great [documentation](https://vuex.vuejs.org/).
+This guide will show you how to use Vuex as you normally would, through the interfaces provided by the Shopware 6 Administration.
+
+## Prerequisites
+
+All you need for this guide is a running Shopware 6 instance, the files and preferably a registered module. 
+Of course you'll have to understand JavaScript and have a basic familiarity with [Vue](https://vuejs.org/) the framework used in the Administration and it's flux library [Vuex](https://vuex.vuejs.org/).
+
+## Creating a store
+The creation of a store is the same as in normal Vuex with the only limitation that all stores have to be `namespaced` in order to prevent collisions with other third party plugins or the Shopware 6 Administration itself.
+
+The following code snippet is the `namespaced` store we will register later through Shopware to the underlying Vuex.
+It is admittedly rather short and has only one variable called `conetent` and a setter for it, but again this all the same as in Vuex. 
+```javascript
+export default {
+    namespaced: true,
+
+    state() {
+        return {
+            // the state we want to keep track of
+            content: ''
+        };
+    },
+
+    mutations: {
+        // a mutation to change the state
+        setContent(state, content) {
+            state.content = content;
+        },
+    }
+};
+```
+
+## Creating the component to use the store
+
+Now that we've created our `namespaced` store, we need to register it to a component to use it.
+This is done through the Shopware object as seen below in the `beforeCreate` [Vue lifecycle hook](https://vuejs.org/v2/guide/instance.html#Lifecycle-Diagram).
+The Shopware object also makes the native Vuex helper functions like [`mapState`](https://vuex.vuejs.org/guide/state.html#the-mapstate-helper), [`mapGetters`](https://vuex.vuejs.org/guide/getters.html#the-mapgetters-helper), [`mapMutations`](https://vuex.vuejs.org/guide/mutations.html#committing-mutations-in-components) and [`mapActions`](https://vuex.vuejs.org/guide/actions.html#dispatching-actions-in-components) available.
+The `namespaced` store itself can be accessed through the `Shopware.State.get()` function.
+Learn more about the Shopware object [here](./the-shopware-object.md).
```

Comment:
As said, I'd reference the Shopware object a bit earlier, and remove it from this line.